fdbclient/ThreadSafeTransaction.cpp

```diff
@@ -401,32 +401,47 @@ ThreadResult<RangeResult> ThreadSafeTransaction::readBlobGranules(const KeyRange
                                                                   Optional<Version> readVersion,
                                                                   ReadBlobGranuleContext granule_context) {
 	// FIXME: prevent from calling this from another main thread!
```

Comment:
This method should not be used and thus should be removed or throw unsupported operation exception. The same for DLTransaction::readBlobGranules. There is option bypassMultiversionApi, but we are going to limit its uses to external client calls.

Reply:
It is currently the case that it would, but there are plans to change that

Reply:
Ok. I changed it to return unsupported_operation and these tests still pass